actionbutton/actionbutton.js

```diff
@@ -42,15 +43,22 @@ function onRecord(row, mappings) {
     if (!row.hasOwnProperty(column)) {
       throw new Error(`Need a visible column named "${column}". You can map a custom column in the Creator Panel.`);
     }
+    let btns = row[column]
+    // If only one action button is defined, put it within an Array
+    if (!btns.length) {
```

Comment:
Can you add a comment here that here we actually are testing if the `length` is in the object, not actually the length of an array. Or maybe it is better to use `Array.isArray` method.